tools/RAiDER/demdownload.py

```diff
@@ -56,28 +57,29 @@ def download_dem(lats, lons, outLoc=None, save_flag='new', checkDEM=True,
         return hgts
 
     # Specify filenames
+    if verbose:
+        print('Getting the DEM')
+        import time
```

Comment:
Just import it at the top of the file, it's a std library package, it will always exist.

Reply:
pushed the change